types/three/examples/jsm/webxr/XRHandOculusMeshModel.d.ts

```diff
@@ -0,0 +1,13 @@
+import { Group, Object3D } from './../../../src/Three';
+
+import { HandModelOptions, XRHandModel } from './XRHandModelFactory';
+
+export class XRHandOculusMeshModel {
+    controller: Group;
+    handModel: XRHandModel;
+    bones: Object3D[];
+
+    constructor(handModel: XRHandModel, controller: Group, path: string, handedness: any, options: HandModelOptions);
```

Comment:
I think `handedness` is either `left` or `right` based on https://github.com/mrdoob/three.js/blob/dev/examples/jsm/webxr/XRHandOculusMeshModel.js#L15 (for `XRHandPrimitiveModel` as well).